### ISSUE TITLE
Since Elixir 1.15 :observer and :debugger need to be mentioned in the `mix.exs` to work in iex

### ISSUE THREAD

**Issue by UweKrause** [elixir-lang/elixir-lang.github.com#1712]
In the (excellent!) getting started guide, in chapter [Dynamic Supervisors](https://elixir-lang.org/getting-started/mix-otp/dynamic-supervisor.html) in section [Observer](https://elixir-lang.org/getting-started/mix-otp/dynamic-supervisor.html#observer), the reader is encouraged to try the `:observer` within the iex, using `iex> :observer.start`.

With Elixir version 1.15, there is a problem with `:observer.start()` in combination with iex, as discussed in the [elixirforum](https://elixirforum.com/t/cannot-start-observer-undefinedfunctionerror-function-observer-start-0-is-undefined/56642).
A working solution, mentioned as [answer in the forum](https://elixirforum.com/t/cannot-start-observer-undefinedfunctionerror-function-observer-start-0-is-undefined/56642/6), is to add `:observer`, `:wx` and `:runtime_tools` in the `:extra_applications` in the file `mix.exs`.

The getting started guide already has a note about problems with `:observer.start()`. I would propose to add another hint to the getting started guide. I would suggest something myself, but since I don't know how fast this problem might be solved, I am not sure if it would be good to document this workaround.

Using the `:observer` in iex is also mentioned in the [Debugging Chapter](https://elixir-lang.org/getting-started/debugging.html#observer) earlier.

Additionally, `:debugger` is also mentioned in the getting started guide chapter about [debugging](https://elixir-lang.org/getting-started/debugging.html#debugging).
This can also be fixed the same way (adding `:debugger` to the `extra_applications` list).

**Comment by josevalim** [comment]
The guides have been moved to Elixir, and I have fixed this there, but I ported here for now until we officially point to the new docs!

**Comment by UweKrause** [comment]
Cool!
I have seen that it got fixed in the guide at one place (chapter "Getting started", Section "Debugging", Subsection "[Observer](https://elixir-lang.org/getting-started/debugging.html#observer)".

But it still is the old text in chapter "Mix and OPT", Section "Dynamic supervisors", Subsection "[Observer](https://elixir-lang.org/getting-started/mix-otp/dynamic-supervisor.html#observer)".

I am also glad that I did not make a CR myself, since you (as someone who is capable of writing this guide) decided to tell the reader another workaround. I (as someone who barely can follow the guide) would have told the reader to just add those dependencies in the `mix.exs`.

Thanks for the guide, it really explains complex stuff in a digestible way :)

**Comment by josevalim** [comment]
Would you like to backport the changes to the Mix guides? Perhaps it needs to be done here and in the Elixir repo. :)

**Comment by UweKrause** [comment]
Sure, I could!
I just dont know which is the correct repo.
I followed the link at the bottom of the page and arrived in this repo.

**Comment by josevalim** [comment]
Yes, this repo is one of them. The other is elixir-lang/elixir. The files will have the same names.

**Comment by UweKrause** [comment]
I am really sorry to bother you again with this really minor stuff, but I noticed that the current solution is not really working.
In my PR, I just copied from "getting started" guide to "mix otp" guide.

![image](https://github.com/elixir-lang/elixir-lang.github.com/assets/16082815/5b25ea5b-1cb5-4859-8e6c-78d47329fdd0)

What would be the right way?
Remove the `> #### Missing dependencies {: .warning}` from both (actually four cause of two repos) places?
I could add another two PRs to do the dirty work :)
